server/controllers/store.js

```diff
@@ -1,25 +1,23 @@
-import express from 'express';
-
+/* eslint-disable class-methods-use-this */
```

Comment:
Please remove the eslint-disable here. Look for a solution for this eslint error

Reply:
error solve and resolved.